### PR TITLE
Update guides.md

### PR DESCRIPTION
**Pull request by the29a**
Autostart pool and virtual machines without HA

### CODE REVIEW

**Comment by olivierlambert** [comment]
Hi,

You can do that by actually just enable a toggle in Xen Orchestra (VM view/advanced). If we decide to add this in the doc, we should probably add both ways. Would you be able to add the XO screenshots for that?

**Comment by the29a** [comment]
Yes, i can add screenshots. But i dont find, where i can enable autostart for pool. I looked over or there is no such possibility  do it via Xen Orchestra.

**Comment by olivierlambert** [comment]
Because the toggle is automatically setting the pool :wink:

**Comment by the29a** [comment]
Ok. How i can upload screenshot?

**Comment by olivierlambert** [comment]
You need to add the image in the `assets` folder and use Markdown to display it.

**Comment by the29a** [comment]
Update commit. I added a screenshot, but I'm not sure I did all right.

File: docs/guides.md [hunk 1]
```diff
@@ -483,3 +483,35 @@ Example with 4 GiB:
...
+4. Enable autostart for each virtual machine with the UUID found: 
+`# xe vm-param-set uuid=021fe476-dc06-23a7-b60e-092a3c286f70 other-config:auto_poweron=true`
+
+5. Checking the output 
+`# xe vm-param-list uuid=021fe476-dc06-23a7-b60e-092a3c286f70 | grep other-config`
+
+#### Xen Orchestra
+
+In Xen Orchestra we can just enable a toggle in VM view. VM toggle in XO is automatically setting the pool.
+![](https://github.com/the29a/xcp-ng-org/blob/master/assets/img/screenshots/autopoweron1.png)
```
Comment (olivierlambert) [inline docs/guides.md:517]: You need to put the file inside the `asset` folder directly, and the link should be:

```
![](./assets/autopower1.png)
```

**Review by olivierlambert — CHANGES_REQUESTED** [review]

File: docs/guides.md (hunk 1 above)
Comment (the29a) [reply docs/guides.md]: Fixed. Kind of.

Comment (olivierlambert) [reply docs/guides.md]: Okay so you need to add the image in the PR. Now there's only the .md file modified.

File: docs/guides.md [hunk 2]
```diff
@@ -483,3 +483,35 @@ Example with 4 GiB:
 Do not mess the units and make sure to set the same value as base value and as max value.
 
 Reboot to apply.
+
+### Autostart pool and virtual machines without HA
+
+We have two ways to sovle problem: CLI and Xen Orchestra.
```
Comment (stormi) [inline docs/guides.md:489]: typo: solve

Comment (the29a) [reply docs/guides.md]: Thanks. Fixed.

File: docs/guides.md (hunk 1 above)
Comment (the29a) [reply docs/guides.md]: https://github.com/xcp-ng/xcp-ng-org/pull/76

**Comment by olivierlambert** [comment]
I would put XO first because it's the easiest way, then the CLI. But you did already some work, so if you are OK I'll merge this and I'll do the next modification after that :+1:

**Comment by the29a** [comment]
Yes, thats ok.

**Comment by olivierlambert** [comment]
Thanks for your contribution @the29a 
Our doc is up to date here: https://xcp-ng.org/docs/guides.html#autostart-vm-on-boot

**Comment by the29a** [comment]
You are welcome.  👍🏻

**Pull request merged**